### PR TITLE
waitings for selenium added

### PR DESCRIPTION
**Pull request by StevenMaMei**

### CODE REVIEW

File: test/buy-tshirt.spec.ts [hunk 1]
```diff
@@ -17,13 +17,8 @@ describe('Buy a t-shirt', () => {
...
-  beforeEach(() => {
-    jasmine.DEFAULT_TIMEOUT_INTERVAL = 120000;
-  });
-
   it('then should be bought a t-shirt', async () => {
     await browser.get('http://automationpractice.com/');
-    await(browser.sleep(5000));
 
     await menuContentPage.goToTShirtMenu();
     await(browser.sleep(3000));
```
Comment (holgiosalos) [inline test/buy-tshirt.spec.ts:24]: all the sleeps of this test class should be removed.

**Review by holgiosalos — COMMENTED** [review]
Please fix the line 58, according to my comment on your last PR:

The idea of the POM is to model the business behavior of an app component, however, the expectations or validations should remain in the test classes. That being said, the right pattern here should be to have a method to retrieve the orderConfirmation from the summary and compare that confirmation with the expected message.

**Comment by StevenMaMei** [comment]
> Please fix the line 58, according to my comment on your last PR:
> 
> The idea of the POM is to model the business behavior of an app component, however, the expectations or validations should remain in the test classes. That being said, the right pattern here should be to have a method to retrieve the orderConfirmation from the summary and compare that confirmation with the expected message.

I changed the validation to be within the test class

File: test/buy-tshirt.spec.ts [hunk 2]
```diff
@@ -50,6 +50,7 @@ describe('Buy a t-shirt', () => {
     await bankPaymentPage.confirmOrder();
     await(browser.sleep(3000));
 
-    orderSummaryPage.confirmIfTheOrderWasSuccesful();
+    await expect(orderSummaryPage.getConfirmationText())
+    .toBe('Your order on My Store is complete.');
```
Comment (holgiosalos) [inline test/buy-tshirt.spec.ts]: Please fix the indent.

**Review by holgiosalos — APPROVED** [review]
Please fix the indent and then you can merge this changes. great work!

**Pull request merged**